packages/api/models/PortfolioDraftSummary.ts

```diff
@@ -1,7 +1,17 @@
+import { ApplicationStep } from "./ApplicationStep";
 import { BaseDocument } from "./BaseDocument";
+import { FundingStep } from "./FundingStep";
+import { PortfolioStep } from "./PortfolioStep";
 import { ProvisioningStatus } from "./ProvisioningStatus";
 
 export interface PortfolioDraftSummary extends BaseDocument {
   status: ProvisioningStatus;
-  num_portfolio_managers: number;
+  portfolio_name?: string;
+  portfolio_step?: PortfolioStep;
+  num_portfolio_managers?: number;
+  funding_step?: FundingStep;
+  num_task_orders?: number;
+  application_step?: ApplicationStep;
+  num_applications?: number;
+  num_environments?: number;
 }
```

Comment:
These are too many fields for the PortfolioDraftSummary, per the spec. Any reason why they're here? Other interfaces later extend this interface to add some of these fields. This should have the exact items shown in the spec for this type (minus ones already specified in `BaseDocument`.

Reply:
Specifcally, `x_step` gets added on the `PortfolioDraft` interface (which extends `PortfolioDraftSummary`). We _must not_ return those fields (or imply they are undefined/null) for a `PortfolioDraftSummary` object.

And if `portfolio_name` is part of the spec, it probably shouldn't be optional. This could be a breaking API change if we ever except a `PortfolioDraftSummary` as input though.

Reply:
@kylelaker-ccpo Agreed.  Many of these don't belong in the PD Summary object.  It should be just enough to support what's displayed for each card of the _My Portfolios_ page.

Reply:
Agreed, I felt the same and should have referenced the API spec. Updated accordingly.